### ISSUE TITLE
add installed tools' binaries to PATH env var

### ISSUE THREAD

**Issue by 2bndy5** [cpp-linter/clang-tools-pip#10]
It might be helpful to add the path to the installed binaries to the OS's PATH env var. Maybe add a CLI flag/switch that performs this?

**Comment by shenxianpeng** [comment]
How to do that? current there is `-d DIRECTORY` option that allows the user to install binaries to the specified path. 
Then add this path to OS's PATH env var?
If the user does not specify a path, the binaries will be installed on the path `~/.local/bin/` on Linux

**Comment by 2bndy5** [comment]
After giving this more thought, we can probably rely on other mechanism to do this (ie the user can do this manually).

I noticed that this isn't needed for working within a python venv because `sys.executable` resolves to the venv's "bin" folder (or "Scripts" on windows).

How I imagined this to be implemented:
```py
import os
import subprocess
import platform

set_env_cmd = "export" if platform.system().lower() == "linux" else "set"  # does not account for macOS
PATH = os.getenv("PATH").split":"
if "directory" not in PATH:
    PATH.append("directory")
    subprocess.run([set_env_cmd, f"PATH={':'.join(PATH)}")
```

Apparently modifying the env vars from within python interpreter is discouraged in the [`os` module's API](https://docs.python.org/3/library/os.html#os.environ). Also note that the `export` command only lasts for as long as the shell is active (all changes revert when terminal is closed).
